agb/src/sound/mixer/hw.rs

```diff
@@ -0,0 +1,92 @@
+use crate::memory_mapped::MemoryMapped;
+
+// Once we have proper DMA support, we should use that rather than hard coding these here too
+const DMA1_SOURCE_ADDR: MemoryMapped<u32> = unsafe { MemoryMapped::new(0x0400_00bc) };
+const DMA1_DEST_ADDR: MemoryMapped<u32> = unsafe { MemoryMapped::new(0x0400_00c0) };
+const _DMA1_WORD_COUNT: MemoryMapped<u16> = unsafe { MemoryMapped::new(0x0400_00c4) }; // sound ignores this for some reason
+const DMA1_CONTROL: MemoryMapped<u16> = unsafe { MemoryMapped::new(0x0400_00c6) };
+
+const DMA2_SOURCE_ADDR: MemoryMapped<u32> = unsafe { MemoryMapped::new(0x0400_00ca) };
```

Comment:
These are not right according to https://mgba-emu.github.io/gbatek/#gbadmatransfers
Recomend calculating these, for instance
```rust
const fn dma_source_address(dma: usize) -> usize {
    0x0400_00B0 + 0x0C * dma
}
const fn dma_dest_address(dma: usize) -> usize {
    0x0400_00B4 + 0x0C * dma
}
const fn dma_word_count_address(dma: usize) -> usize {
    0x0400_00B8 + 0x0C * dma
}
const fn dma_control_address(dma: usize) -> usize {
    0x0400_00B0 + 0x0C * dma
}```